core/src/main/kotlin/io/customer/sdk/core/di/SDKComponent.kt

```diff
@@ -23,6 +24,9 @@ object SDKComponent : DiGraph() {
     // Static map to store all the modules registered with the SDK
     val modules: MutableMap<String, CustomerIOModule<*>> = mutableMapOf()
 
+    // Android specific dependencies
```

Comment:
should this be in `AndroidSDKComponent`?

Reply:
```suggestion
    // Android component dependencies
```